charts/seed-operatingsystemconfig/original/templates/scripts/_health-monitor.sh

```diff
@@ -136,6 +148,8 @@
         echo "Start kubernetes health monitoring for $component"
         if [[ $component == "docker" ]]; then
```

Comment:
What about the Docker monitoring script? Do we still need it even if we containerd is enabled?

Reply:
For now, It is still possible to work with Docker. When you choose that docker monitor service (using the docker monitor script)